.eslintrc

```diff
@@ -31,7 +31,6 @@
         "prefer-template": "error",
         "comma-spacing": ["error", { "before": false, "after": true }],
         "space-before-function-paren": ["error", "never"],
-        "space-before-blocks": ["error"],
```

Comment:
why removed?

Reply:
It was duplicated @yukie1412 . Please check line 30.